scripts/l10n.ts

```diff
@@ -442,7 +442,10 @@ async function run(argv: yargs.ArgumentsCamelCase<HandlerArgv>) {
           }
           if (!missingTranslations[localeCode][packageInfo.key]) {
             missingTranslations[localeCode][packageInfo.key] = {
-              path: localePath.replace(workspaceRoot, '').slice(1), // Remove leading slash
+              // prettier-ignore
```

Comment:
Prettier would put every function call on a new line.
Given that the comments apply in the way I've formatted, I've added a `prettier-ignore` as it looks more readable this way. 🤔